README.md

```diff
@@ -31,3 +37,15 @@ This builds on [ASP.NET Core MVC tag helpers for GOV.UK Design System](https://g
 - [Override property values](docs/umbraco/override-property-values.md)
 - [Umbraco unit testing](docs/umbraco/unit-testing.md)
 - [Use SASS for CSS](docs/aspnet/sass.md)
+
+Umbraco support is published on NuGet as [ThePensionsRegulator.GovUk.Frontend.Umbraco](https://www.nuget.org/packages/ThePensionsRegulator.GovUk.Frontend.Umbraco)
+
+![ThePensionsRegulator.GovUk.Frontend.Umbraco on nuget.org](https://img.shields.io/nuget/vpre/ThePensionsRegulator.GovUk.Frontend.Umbraco)
```

Comment:
Will this badge be increased to v3.0 automatically? It sayd 2.0 at the moment.

Reply:
Yes